integ-test/src/test/resources/clickbench/queries/q39.ppl

```diff
@@ -0,0 +1,11 @@
+/*
+SELECT URL, COUNT(*) AS PageViews
+FROM hits WHERE CounterID = 62 AND EventDate >= '2013-07-01' AND EventDate <= '2013-07-31'
+AND IsRefresh = 0 AND IsLink <> 0 AND IsDownload = 0
+GROUP BY URL ORDER BY PageViews DESC LIMIT 10 OFFSET 1000;
```

Comment:
@penghuo  can you confirm this transfer needs `streamstats` command?

```
| sort _time| streamstats count as row_number| where row_number > 1000 AND row_number <= 1010
```

Reply:
No. `streamstats count as row_number` should be similar to SQL `COUNT(*) OVER (ORDER BY _time)` with PRECEDING to CURRENT window frame.